### ISSUE TITLE
Fix WCAG 1.3.1: Info and Relationships issues

### ISSUE THREAD

**Issue by juliatufts** [scottrigby/basekamp-next#4]
Fix the following issues related to WCAG 1.3.1: Info and Relationships, see also: [full accessibility report](https://github.com/basekamp/basekamp-next/wiki/Accessibility-Evaluation-Report-2026).

## Entire sample

Observations:

- [x] Not sure if Basekamp logo up top should be included in nav, probably yes?
  - [x] addressed in https://github.com/scottrigby/basekamp-next/issues/4#issuecomment-4020648869
- [x] both banner and nav are present and pretty much lead to the same section
  - question for reviewer in https://github.com/scottrigby/basekamp-next/issues/4#issuecomment-4026473322
  - [x] answered in https://github.com/scottrigby/basekamp-next/issues/4#issuecomment-4028105473

## About

No observations added

## Projects

- [x] Projects should be a list
  - #22

## Events

- [x] Events should be a list
  - #22

## Contact

No observations added

## Project page (PAW)

- [x] Schedule should be table
  - #21
- [x] Some entries are double starred but no legend is provided to determine what that means
  - #21

## Event page (Homework)

- [x] "Broken City Lab" is marked strong, not sure if that's best semantically
  - question for reviewer in https://github.com/scottrigby/basekamp-next/issues/4#issuecomment-4026394577
  - [x] answered in https://github.com/scottrigby/basekamp-next/issues/4#issuecomment-4028089293

**Comment by scottrigby** [comment]
> * Not sure if Basekamp logo up top should be included in nav, probably yes?

According to https://www.w3.org/WAI/WCAG22/Techniques/html/H101 the current markup creates an implicit "banner" role for our `<header>`, which contains our logo. This seems to fit their description of the banner role:
> a region that typically contains the site's logo, name, and other persistent site-wide content at the top of a page

**Comment by scottrigby** [comment]
> * "Broken City Lab" is marked strong, not sure if that's best semantically

Given that the content files are now markdown - as a choice for making the content maintainable - and in this example, these projects in `<strong>` are the lead in to sentences (rather than definitions), should `<strong>` (important) be enough here for accessibility?

---

Since this is part of the page content, I do think a full content review would be good. These projects and events happened over many years. Some of the pages are essentially straight from the paper and email announcements during those times, while others were written as web content. The format of the content area could be reviewed for semantic readability, and format consistency.

@juliatufts should a content review like this be part of the accessibility project, or a separate follow-up?

**Comment by scottrigby** [comment]
> * both banner and nav are present and pretty much lead to the same section

@juliatufts could you help me understand what this means? My first thought is "banner" refers to the `<header>` (since it's a direct child of `<body>` so implicitly gets `banner` role, per https://www.w3.org/WAI/WCAG22/Techniques/html/H101)? Currently, `<nav>` is inside `<header>` - do screen readers consider this a duplicate? If so, should we move `<nav>` outside of `<header>`?

**Comment by juliatufts** [comment]
> > * "Broken City Lab" is marked strong, not sure if that's best semantically
> 
> Given that the content files are now markdown - as a choice for making the content maintainable - and in this example, these projects in `\<strong\>` are the lead in to sentences (rather than definitions), should `\<strong\>` (important) be enough here for accessibility?
> 
> Since this is part of the page content, I do think a full content review would be good. These projects and events happened over many years. Some of the pages are essentially straight from the paper and email announcements during those times, while others were written as web content. The format of the content area could be reviewed for semantic readability, and format consistency.
> 
> [@juliatufts](https://github.com/juliatufts) should a content review like this be part of the accessibility project, or a separate follow-up?

This can be a separate follow up. I don't think it's a violation per se, but could be improved upon.

**Comment by juliatufts** [comment]
> > * both banner and nav are present and pretty much lead to the same section
> 
> [@juliatufts](https://github.com/juliatufts) could you help me understand what this means? My first thought is "banner" refers to the `<header>` (since it's a direct child of `<body>` so implicitly gets `banner` role, per https://www.w3.org/WAI/WCAG22/Techniques/html/H101)? Currently, `<nav>` is inside `<header>` - do screen readers consider this a duplicate? If so, should we move `<nav>` outside of `<header>`?

I think this is a non-issue. I was referring to when you navigate by landmark using VoiceOver, clicking on either Banner or Navigation will both lead to the top of the Banner (as expected). I was questioning whether the semantic grouping was helpful in this case, and while slightly redundant, I think it's fine. Yes `<header>` has implicit `banner` role.

**Comment by scottrigby** [comment]
OK great, updating checklist above and closing this issue 👍
